clickhouse-core/src/main/scala/xenon/clickhouse/spec/TableSpec.scala

```diff
@@ -27,11 +27,11 @@ case class TableSpec(
   uuid: String,
   engine: String,
   is_temporary: Boolean,
-  data_paths: Array[String],
+  data_paths: List[String],
   metadata_path: String,
   metadata_modification_time: LocalDateTime,
-  dependencies_database: Array[String],
-  dependencies_table: Array[String],
```

Comment:
the key point is equals? if so, does Seq work?

Reply:
Seq is not Serializable